bot/exts/moderation/infraction/management.py

```diff
@@ -150,6 +150,10 @@ async def infraction_edit(
             request_data['expires_at'] = None
             confirm_messages.append("marked as permanent")
         elif duration is not None:
+            now_datetime = datetime.now(duration.tzinfo) if duration.tzinfo else datetime.utcnow()
```

Comment:
Notice that `datetime.utcnow()` is actually timezone unaware (yeah, I know), which could become a problem since you wouldn't be able to compare the timezone-unaware `now_datetime` to the timezone-aware `duration`. In practice this will never happen since `duration` should always have timezone information.

Furthermore, even if `duration`'s timezone could be something other than UTC, there's no need to take it into account when creating `now_datetime`, since comparing two datetimes in different timezones will cause the library to make the appropriate conversions.

So all in all it's fine to always have `now_datetime = datetime.now(timezone.utc)` when `timezone` is imported from `datetime`. Or just use `arrow.utcnow()` which is timezone aware.